dpnp/backend/mkl_wrap_rng.cpp

```diff
@@ -70,8 +65,8 @@ void mkl_rng_uniform(void* result, long low, long high, size_t size)
 
 }
 
-template void mkl_rng_gaussian<double>(void* result, size_t size);
-template void mkl_rng_gaussian<float>(void* result, size_t size);
+template void mkl_rng_gaussian<double>(void* result, double mean, double stddev, size_t size);
```

Comment:
Could you please align perametrization with new approach?
Example:
https://github.com/IntelPython/dpnp/blob/fb5e82c850bade4e23457e19b1c7826e4c761019/dpnp/backend/custom_kernels_linalg.cpp#L72-L80

Or you could look into PR #151 
Also, one more example might be last commit in PR #150